dev/AppNotifications/AppNotificationBuilder/AppNotificationBuilderUtility.h

```diff
@@ -145,16 +120,16 @@ inline std::wstring EncodeXml(winrt::hstring const& value)
 // https://github.com/CommunityToolkit/WindowsCommunityToolkit/blob/rel/7.1.0/Microsoft.Toolkit.Uwp.Notifications/Toasts/ToastArguments.cs#L389inline
 inline std::wstring Decode(std::wstring const& value)
```

Comment:
I hope you have a nice unit test for this

Reply:
Similarly something for decoding

Reply:
I have multiple tests covering cases.